lib/resolveTaskFn.js

```diff
@@ -2,14 +2,17 @@ import { redBright, dim } from 'colorette'
 import execa from 'execa'
 import debug from 'debug'
 import { parseArgsStringToArgv } from 'string-argv'
+import pidTree from 'pidtree'
```

Comment:
These imports are not in alphabetical order, but maybe we should setup an ESLint rule for that.